src/minimax_server/router.clj

```diff
@@ -0,0 +1,18 @@
+(ns minimax-server.router
+  (:import [java.util.function Function]
+           [scarvill.httpserver.response ResponseBuilder])
+  (:require [clojure.string :as str]))
+
+(defn response-with-body [body]
+  (.build (.setBody (new ResponseBuilder) (byte-array (map byte (str body))))))
```

Comment:
Since your `ResponseBuilder` object has a fluent interface (if I remember correctly) I think you could also just use the threading macro, but then you would possibly need to add type hints to every call, which isn't ideal.